README.md

```diff
@@ -126,41 +89,9 @@ Score review and the "Try Again" button:
 
 ### Free-form questions
 
-Free-form questions are represented by an `<answer>` element. The
-answer element supports the following attributes:
+Free-form questions are represented by a "Long Answer" component. 
 
-* `name` - Sets the name of the question. The name can be used to
-  refer to the question when displaying the answer in another block.
-  The name is not visible to the student. Should be unique within a
-  course.
-* `weight` - The weight is used when computing total grade/score of
-  the mentoring block. The larger the weight, the more influence this
-  question will have on the grade. Value of zero means this question
-  has no influence on the grade (float, defaults to `1`).
-* `min_characters` - The minimum length of the answer. If the answer
-  contains less than the specified number of characters, the answer is
-  considered invalid and the student will not be allowed to submit the
-  answer (integer; defaults to `0`).
-* `read_only` - Whether the answer should be rendered read-only. This
-  only makes sense when displaying an answer to a previously answered
-  question. The answer is rendered as a HTML quote instead of a
-  textarea element (boolean; defaults to `false`).
-
-It can also have a `<question>` element containing a paragraph of non-formatted plain text.
-
-#### Example
-
-Here is a simple example of a free-form question:
-
-```xml
-<mentoring url_name="goal_definition" followed_by="getting_feedback" weight="20">
-    <answer name="goal" weight="10">
-        <question>What is your goal?</question>
-    </answer>
-</mentoring>
-```
-
-Screenshot before answering the question:
+Example screenshot before answering the question:
 
 ![Answer Initial](https://raw.githubusercontent.com/edx-solutions/xblock-mentoring/1fb5e3ece6f34b6cf33c956a4fba1d7cbb7349a2/doc/img/answer-1.png)
```

Comment:
You should just be able to do a relative path to the image file, and Github will grab it properly. Using these hashes like this means that if the file is updated later, it will not automatically update the README. I know this isn't really something that's directly in the scope of the PR, but while we're here...